HISTORY.rst

```diff
@@ -25,6 +25,55 @@ unreleased
   are supported. Once registered, the device will show up in the linked IoT Hub and can be interacted with or
   simulated using other `az iot device` commands.
 
+**IoT Central updates**
+
+* Add support for device groups CRUD.
+  - az iot central device-group
+    - az iot central device-group list
+    - az iot central device-group show
+    - az iot central device-group create
+    - az iot central device-group delete
+    - az iot central device-group update
+
+* Add support for device attestation CRUD.
+  - az iot central device attestation
+    - az iot central device attestation show
+    - az iot central device attestation create
+    - az iot central device attestation delete
+    - az iot central device attestation update
+
+* Add support for device/module properties CRUD.
+  - az iot central device list-components
+  - az iot central device show-telemetry-value
+  - az iot central device show-component-telemetry-value
+  - az iot central device properties
+    - az iot central device properties show
+    - az iot central device properties replace
+    - az iot central device properties update
+    - az iot central device properties show-component-properties
```

Comment:
do a separate group for component-properties:
az iot central device properties
az iot central device component-properties
the commands will feel less awk then

Reply:
Sure

Reply:
az iot central device properties component-properties show